Solutions/Microsoft Defender for Office 365/Playbooks/O365_Defender_FunctionAppConnector/readme.md

```diff
@@ -0,0 +1,118 @@
+# Defender for office 365 API Functions App Connector
+
+This Functions App Connector is to connect Defender for office 365 API.
+
+### Authentication methods supported by this connector
+
+* Custom Authentication/Certificate Authentication
+
+### Prerequisites For Defender for office 365 API Functions App Connector
+
+* App Registration on azure active directory is needed , Beacuse this connector uses certificate thumprints for Authentication 
```

Comment:
Correct the spelling 'Beacuse' and 'thumprints'.

Reply:
Done